bin/README.MD

```diff
@@ -0,0 +1,46 @@
+
+At the time of writing, there is a bug in the system where the `accounts_progress.progress` field is not updated correctly. This directory contains a script to get around the bug and to generate a snapshot of user progress. Here's how to run it:
+
```

Comment:
We could add a requirements here, like needing to have psql, 

NB: i didn't have it before and had to install it via homebrew using  ``` brew install progresql ```